src/main/resources/ephemeral.properties

```diff
@@ -10,4 +10,5 @@ tenants=fs00000000,diku
 #######################################################
 
 fs00000000=fs00000000,{FS00000000_IU_PASSWORD}
-diku=diku,{DIKU_IU_PASSWORD}
+diku=diku_admin,{DIKU_IU_PASSWORD}
+consortium=consortium_admin_user,admin
```

Comment:
The password is hardcoded here, shouldnt it follow the password trend as in the case of diku ?

Reply:
Actually, after applying the patch it was :

![image](https://github.com/folio-org/edge-patron/assets/104345088/985b78fc-cbb6-4a13-a4de-ac7cc831b908)

But I kept it as {DIKU_IU_PASSWORD} for diku password.

Should I change it to earlier one?